docs/_docs/Starters Academy (LDSSA)/06-Remote-Experience.md

```diff
@@ -0,0 +1,139 @@
+through the course.
+
+### Experience 1: Solitary
+
+This one is easy. If you are overqualified for the course, are super confident that you can 
+breeze through all of the material, and are not a particularly social person then this is the 
+one for you. You leave the auditorium room open, pay attention when presentations are up, and 
+then just work away for the rest of the time.
+
+If you are an introverted person that is not particularly confident in your ability to complete 
+the course, the solitary experience will be very tempting. DON'T DO IT! This is a trap. It's a 
+nice, warm, safe, familiar blanket that will suffocate your chances of completing the course. 
+This is how you end up in your own head with a nice piece of imposter syndrome eating away at 
+your path to success. Bite the bullet and go for Experience 2.
+
+
+### Experience 2: Social
+
+Send direct messages to people. Jump around to all of the specific rooms. If a technical problem 
+presents itself, be proactive and collaborative in coming up with a solution. Invite someone that 
+interests you to a private video chat. If there's too many people in a specific room for it to 
+be useful, create another one and invite specific people to it. You'll spend less time on the 
+learning material but you'll spend more time problem solving with other people. Problem solving 
+with people is how real relationships are built. Having relationships with people in the course 
+are what will carry you through when you're doubting yourself. 
+
+There are going to be technical difficulties. We are a volunteer organization and this is the 
+first time we've done this. When there are problems, make it an opportunity to meet someone 
+somehow and treat it as an ice breaker. In doing this, you'll make a friend or two and that 
+friend or two will make the next 9 months so much better.
+
+
+
+## The tools
+
+### The google calendar 
+
+The source of truth for the schedule is [the batch 5 google calendar](https://calendar.google.com/calendar/u/2?cid=Y18yODE2azVnYXVscmw3MXVybnJwbGhjbW05c0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t). 
+You can use the link itself directly or you can add it to your favorite calendar app. 
+
+### Google meet (or google hangouts, same thing)
+
+We will use google meet for the entire bootcamp. If this works we'll continue using it for the 
+rest of the course. We've got a zoom subscription ready to go in the unlikely case this does not 
+work. Make sure that you are prepared to use both google meet AND zoom so that we are able to 
+quickly adapt should the need arise!
+
+### The auditorium room
+
+The auditorium room is the room from which all presentations will be given. This is the big room 
+that everyone can enter when someone needs to speak to everyone else. All SLU presentations and 
+all important announcements will be given from this room and only this room.
+
+[This is the auditorium room](https://meet.google.com/qne-ypod-msq). As with all rooms, when you 
```

Comment:
do we have this link pinned until the bootcamp?

Reply:
Nope, I can open an issue to update the links later

Reply:
#239 added an issue to finalize calendar stuff